.travis.yml

```diff
@@ -31,6 +31,14 @@ matrix:
         #- '[ "$TRAVIS_SECURE_ENV_VARS"  = "true" ] || mvn -Dmaven.javadoc.skip=true -Dbasepom.check.skip-basic=true -Dbasepom.check.skip-findbugs=true -Dbasepom.check.skip-pmd=true -Dbasepom.check.skip-checkstyle=true -Ptoolchains        -B verify'
         - mvn -Dmaven.javadoc.skip=true -Dbasepom.check.skip-basic=true -Dbasepom.check.skip-findbugs=true -Dbasepom.check.skip-pmd=true -Dbasepom.check.skip-checkstyle=true -Ptoolchains -B verify
       jdk: oraclejdk8
+    - env:
+        - TESTS=Y
+      script:
+        # Oracle profile disabled until they fix their orai18n JAR artifact on maven.oracle.com repository.
+        #- '[ "$TRAVIS_SECURE_ENV_VARS" != "true" ] || mvn -Dmaven.javadoc.skip=true -Dbasepom.check.skip-basic=true -Dbasepom.check.skip-findbugs=true -Dbasepom.check.skip-pmd=true -Dbasepom.check.skip-checkstyle=true -Djdk9=true -Ptoolchains,oracle -B verify'
```

Comment:
can we factor all this junk out into matrix better?  it sucks to have to copy+paste it around, but I'm not super comfortable with the build matrix to know how to do better.

Reply:
I don't know why we're not just putting these commands in bash scripts and having our matrix just invoke those. This "fit everything on one line" thing makes it really hard to read anything.

Reply:
That makes sense to me.  I have a pretty busy week but I'll take a crack at it sometime unless someone else gets to it first.